### DISCUSSION TITLE
Supported TypeScript Version Warning

### DISCUSSION THREAD

**Discussion by keita-determined** [typescript-eslint/typescript-eslint#7972] [Community Feedback]
Hello, 

I am using 
 - `"eslint": "^8.54.0"`
 - `"@typescript-eslint/parser": "^6.12.0"`
 - `"@typescript-eslint/eslint-plugin": "^6.12.0"`
 - `"typescript": "^5.3.2"`.

[The website](https://typescript-eslint.io/users/dependency-versions/#typescript) says 

> The version range of TypeScript currently supported is >=4.3.5 <5.4.0 || 5.3.0-beta || 5.3.1-rc || 5.4.0-dev.20231117.

However, I get this warning

> =============
> 
> WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.
> 
> You may find that it works just fine, or you may not.
> 
> SUPPORTED TYPESCRIPT VERSIONS: >=4.3.5 <5.3.0
> 
> YOUR TYPESCRIPT VERSION: 5.3.2
> 
> Please only submit bug reports when using the officially supported version.
> 
> =============

and the info does not look look like updated. Is the info correct or am I missing something? 

https://github.com/typescript-eslint/typescript-eslint/blob/810fc8c9a18cb55af144d8945fdf38d841c9d486/packages/typescript-estree/src/parseSettings/warnAboutTSVersion.ts#L9

**Comment by bradzacher** [comment]
We bumped the range kn the docs a little early. We don't yet fully support it.

https://github.com/typescript-eslint/typescript-eslint/issues/7719
